src/app/examples/file/file.component.html

```diff
@@ -0,0 +1,35 @@
+<app-navbar></app-navbar>
+
+<div class="m-10">
+  <h1>File example form</h1>
+
+  <div class="grid grid-cols-2 gap-4">
+    <div>
+      <form #fileInputForm="ngForm" (ngSubmit)="submit()" class="form">
+        <label class="field">
+          <span class="field-label">Pick file</span>
+          <input ngModel name="file" [(ngModel)]="model.file" type="file" #fileInput="ngModel" id="files" />
+          <mz-field-errors [field]="fileInput"></mz-field-errors>
```

Comment:
Would you add red styling to make it clear in the example that it's a field error?